src/app/component/CheckoutOrderSummary/CheckoutOrderSummary.component.js

```diff
@@ -56,7 +62,9 @@ export class CheckoutOrderSummary extends PureComponent {
                 </strong>
                 <strong block="CheckoutOrderSummary" elem="Text">
                     { `${priceString}` }
+                    { subPrice }
```

Comment:
Consider using `renderThing` pattern here and determine whether to render this or not depending on the value you process there. Too lot of args for one function :)

Reply:
There are multiple calls of this helper function with the very different args.
Moved it to a separate component.